sdk/ml/azure-ai-ml/azure/ai/ml/_utils/_registry_utils.py

```diff
@@ -104,7 +104,7 @@ def get_sas_uri_for_registry_asset(service_client, name, version, resource_group
             registry_name=registry,
             body=body,
         )
-        sas_uri = res.blob_reference_for_consumption.credential.sas_uri
+        sas_uri = res.blob_reference_for_consumption.credential.additional_properties["sasUri"]
```

Comment:
Is the SAS URI field moving to inside `additional_properties` or was it a bug all along we were looking in the wrong place?

Reply:
Since the swagger changes, it got changed little bit on the deserialization, everything is autogenerated, one of our E2E test caught this...